store/mongo/datastore_mongo.go

```diff
@@ -167,6 +167,10 @@ func (db *DataStoreMongo) GetDevices(ctx context.Context, skip int, limit int, f
 		}
 	}
 
+	if groupName != "" {
+		queryFilters = append(queryFilters, bson.M{DbDevGroup: groupName}) // TODO - is this the right db-call?
```

Comment:
I think this is the right db call. It works, but it has not been tested along with filters etc. But I do not believe that should cause a problem?

Reply:
What do you mean by 'it has not been tested along with filters'? From the code, `queryFilters` is used with mongo's `$and` operator. I think it's ok.